wis2box-management/wis2box/pubsub/message.py

```diff
@@ -210,6 +212,30 @@ def __init__(self, identifier: str, metadata_id: str, filepath: str,
             }
             self.message['links'].append(link)
 
+        LOGGER.debug(f'Checking for access control (metadata id: {metadata_id})')  # noqa
+        try:
+            oar = Records(DOCKER_API_URL)
+            record = oar.collection_item('discovery-metadata', metadata_id)
+
+            if record['wis2box'].get('has_auth'):
+                LOGGER.debug('Updating message with access control')
+
+                for link in self.message['links']:
+                    if link['href'] == public_file_url:
+                        LOGGER.debug('Adding security object to link')
+                        link['security'] = {
+                            'default': {
```

Comment:
Is the use of `default `consistent with https://raw.githubusercontent.com/OAI/OpenAPI-Specification/3.1.0/schemas/v3.0/schema.yaml#/definitions/Components/properties/securitySchemes:~:text=HTTPSecurityScheme%3A%0A%20%20%20%20type%3A%20object,not%3A%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20enum%3A%20%5Bbearer%5D

Reply:
Yes, one can define 1..n security objects, defined by user-specified keys.  `default` is a convention used by other centres (dk-dmi for example) and is mentioned in the [WIS2 Cookbook](https://wmo-im.github.io/wis2-cookbook/cookbook/wis2-cookbook-DRAFT.html#_publish_a_wis2_notification_message_with_access_control).